library/common/extensions/filters/http/platform_bridge/filter.cc

```diff
@@ -153,8 +154,42 @@ void PlatformBridgeFilter::onDestroy() {
   platform_filter_.instance_context = nullptr;
 }
 
+void PlatformBridgeFilter::dumpState(std::ostream&, int) const {
+  // TODO(junr03): output to ostream - https://github.com/envoyproxy/envoy-mobile/issues/1497.
+  ENVOY_LOG(error, "PlatformBridgeFilter: {} error_response={}", filter_name_, error_response_);
+  if (request_filter_base_) {
+    auto buffer = request_filter_base_->buffer();
+    ENVOY_LOG(
+        error,
+        "  Request Filter state={} on_headers_called={} on_data_called={} on_trailers_called={} "
+        "on_resume_called={} buffer={} stream_complete={}",
+        (request_filter_base_->iteration_state_ == IterationState::Ongoing ? "ongoing" : "stopped"),
+        request_filter_base_->on_headers_called_, request_filter_base_->on_data_called_,
+        request_filter_base_->on_trailers_called_, request_filter_base_->on_resume_called_,
+        (buffer ? fmt::format("{} bytes", buffer->length()) : "no buffer"),
+        request_filter_base_->stream_complete_);
+  } else {
+    ENVOY_LOG(error, "  Request Filter absent");
```

Comment:
"absent" sort of makes it sound like it's _supposed_ to be there, which it isn't necessarily. I'd probably just leave this blank, or if you feel it's still helpful, say something like "No request filter."